tests/get_search.test.ts

```diff
@@ -473,7 +473,19 @@ describe.each([
       .index(emptyIndex.uid)
       .searchGet('', { vector: [1], hybridSemanticRatio: 1.0 })
 
-    expect(response.vector).toEqual([1])
+    expect(response).toHaveProperty('hits')
+    expect(response).toHaveProperty('semanticHitCount')
+    // Those fields are no longer returned by the search response
+    // We want to ensure that they don't appear in it anymore
+    expect(response).not.toHaveProperty('vector')
+    expect(response).not.toHaveProperty('_semanticScore')
```

Comment:
I test that I retrieve the `hits` and `semanticHitCount` fields correctly.
And since the `vector` and `_semanticScore` fields have been removed from the response, I check that I don't retrieve them anymore